### DISCUSSION TITLE
플레이시 캐릭터의 머리카락이 자연스럽게 안 움직입니다.

### DISCUSSION THREAD

**Discussion by sul321** [naverz/zepeto-studio-kor#1229] [5. Q&A]
캐릭터 플레이 하면 캐릭터의 머리카락이 지점토처럼 굳어 보이게 움직입니다. 
이는 어떻게 해결해야 될까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 별도의 스크립트가 적용되지 않았는데도 그렇게 보이는 것이죠?
관련해서 스크린샷이나 영상이 있으면 좋을 것 같습니다.
